crates/ruff_linter/src/rules/eradicate/detection.rs

```diff
@@ -53,6 +56,15 @@ pub(crate) fn comment_contains_code(line: &str, task_tags: &[String]) -> bool {
         return false;
     }
 
+    // Fast path: if the comment starts with two consecutive identifiers, we know it won't parse,
+    // unless the first identifier is a keyword.
+    let mut tokenizer = SimpleTokenizer::starts_at(TextSize::default(), line).skip_trivia();
+    for (a, b) in tokenizer.tuple_windows() {
+        if a.kind == SimpleTokenKind::Name && b.kind == SimpleTokenKind::Name {
+            return false;
+        }
+    }
```

Comment:
The comment mentions that this tests if the comment starts with two consecutive identifiers but the implementation consumes all tokens and tests if there are any consecutive identifiers. 

If it's only the first pair, than I would probably use (I'm not a huge can of `tuple_windows` because it's a performance footgun if the values are expensive to `Clone`. This is not the case here but it's the reason why I try to avoid it in general).

```rust
if let (Some(a), Some(b)) = (tokenizer.next(), tokenizer.next()) {
	if a.kind == SimpleTokenKind::Name && b.kind == SimpleTokenKind::Name {
	  return false;
	}
}
```

Reply:
What if we have 

```python
# a
# b
```

Or does this only ever run over a single comment

Reply:
It’s in draft for this reason — haven’t finished the PR!

(This only ever runs over a single own-line comment.)